Routes/products.route.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const { Product } = require("../Models/product.model.js");
+const router = express.Router();
+
+router.route("/")
+.get(async (req, res) => {
+  try {
+    const products = await Product.find({});
```

Comment:
If you trying to find al the products you can simply no arguments like this

```
Product.find()
```

Reply:
Yes, I have fixed this issue